### ISSUE TITLE
Carousel with .carousel-slider only shows half of the image

### ISSUE THREAD

**Issue by Thanood** [Dogfalo/materialize#2741]
Here is a plunk using the carousel from your website:
http://plnkr.co/edit/6cJYKdtniiTUfLib47EA?p=preview

The image is 400px in height but only shows about the half of it.
I guess it is because of: https://github.com/Dogfalo/materialize/blob/master/js/carousel.js#L97

``` javascript
alignment += 'translateY(' + (view[0].clientHeight - item_width) / 2 + 'px)';
```

Setting `translateY(0)` in chrome debugger fixes this. Also, there is no `translateY` set in the example on Materialize website.

As you might have guessed, images with same height/width display correctly.

**Comment by sapkra** [comment]
Use this an get lucky. :)

`$('.carousel').carousel({full_width: true});`

**Comment by Thanood** [comment]
Yeah, why not. :smile: 

Anyway, nice workaround but I guess this could still get fixed. I'm just not sure if my assumption is correct. If I were sure, I'd commit a PR. :smile:

**Comment by acburst** [comment]
Fixed in bb5dd21

**Comment by ajomadlabs** [comment]
I wanted to have carousel for cards . Is there any way to obtain it
